src/components/navbar/Navbar.js

```diff
@@ -166,18 +171,30 @@ const Navbar = () => {
             />
           </div>
           <div className="search-box">
-            <input type="text" placeholder="Search for :" />
+            <input type="text" placeholder="Search for :" onChange={(e) => setSearchProduct(e.target.value)} />
             <div className="search-icon-img">
               <img src="/images/search-light.png" alt="search" />
             </div>
           </div>
           <div className="search-text">
-            Find your product with fast search. Enter some keyword such as
-            dress, jacket etc.
+            {searchProduct.length < 2 ? <span>Find your product with fast search. Enter some keyword such as
+              dress, jacket etc.</span> : (
+              <>
+                {
+                  products.filter((filterData) => filterData.title.toLowerCase().includes(searchProduct.toLowerCase()) ? filterData : null)
+                    .map((items) => {
+                      return (
+                        <SearchCard pID={items._id} imgSrc={items.imgOne} title={items.title} price={items.price} />
```

Comment:
Here also, you can send the entire object and destructure it in SearchCard component.

Reply:
i will try to impliment it